katharsis-client/src/main/java/io/katharsis/client/internal/BaseResponseDeserializer.java

```diff
@@ -15,6 +15,7 @@
 import io.katharsis.dispatcher.controller.resource.ResourceUpsert;
 import io.katharsis.jackson.exception.JsonDeserializationException;
 import io.katharsis.queryParams.QueryParams;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.queryParams.QueryParams'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)